gdal-src/Cargo.toml

```diff
@@ -0,0 +1,314 @@
+[package]
+name = "gdal-src"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+proj-sys = { version = "0.23.2", features = ["bundled_proj"] }
+libsqlite3-sys = { version = "0.27.0", features = ["bundled"], optional = true }
+hdf5-src = { version = "0.8.1", optional = true, features = ["hl"] }
+hdf5-sys = { version = "0.8.1", optional = true, features = ["static"] }
+netcdf-src = { version = "0.3.2", optional = true }
+pq-src = { version = "0.1", optional = true }
+curl-sys = { version = "0.4.70", features = ["static-curl"], optional = true }
+libz-sys = { version = "1.1.15", features = ["static"], optional = true }
+
+[build-dependencies]
+cmake = "0.1.50"
+
+[features]
+default = []
```

Comment:
It's up to discussion which drivers should be build by default as cargo feature flags. We cannot easily say build everything and allow users to opt out as cargo feature flags are additive.

Reply:
I guess NetCDF and Zarr could be interesting.

Reply:
The zarr driver so currently not not supported by the bundling. It would require figuring out how to bundle all the dependencies for that driver as well. 

Netcdf is supported, as there is already a rust crate which provides bundling support and exposes the relevant information. Therefore it should be possible to enable it by default.